### PR TITLE
[fundamental] Add workflow to clean runtime disk

### PR DESCRIPTION
**Pull request by zhengfeiwang**
# Description

Add workflow [Clean runtime disk](https://github.com/microsoft/promptflow/actions/workflows/clean-runtime-disk.yml) to create runs that will cleanup runtime disk in test/example runtime in canary/production (East US) regions.

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: scripts/runtime_mgmt/clean-disk-flow/clean.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import subprocess
+from promptflow import tool
+
+bash_script = """
+#!/bin/bash
+
+app_dir="/service/app"
+
+for process_dir in $(find "$app_dir" -mindepth 1 -maxdepth 1 -type d); do
+    if [ "$process_dir" != "/service/app/flow_pids" ]; then
```
Comment (crazygao) [inline scripts/runtime_mgmt/clean-disk-flow/clean.py:10]: use $appdir？

Comment (zhengfeiwang) [reply scripts/runtime_mgmt/clean-disk-flow/clean.py]: good catch, updated

Comment (zhengfeiwang) [reply scripts/runtime_mgmt/clean-disk-flow/clean.py]: good catch, updated.

File: scripts/runtime_mgmt/manage_runtime.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+def main(args: argparse.Namespace):
+    ml_client = init_ml_client(
+        subscription_id=args.subscription_id,
+        resource_group_name=args.resource_group_name,
+        workspace_name=args.workspace_name,
+    )
+    pfs_runtime_helper = PFSRuntimeHelper(ml_client=ml_client)
+
+    # print("deleting runtime...")
```
Comment (crazygao) [inline scripts/runtime_mgmt/manage_runtime.py:76]: direct delete this part or make a control argument?

Comment (zhengfeiwang) [reply scripts/runtime_mgmt/manage_runtime.py]: removed this file in this PR

File: scripts/runtime_mgmt/clean-disk-flow/flow.dag.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+id: clean_runtime_disk
```
Comment (wangchao1230) [inline scripts/runtime_mgmt/clean-disk-flow/flow.dag.yaml:1]: remove id & name

**Review by wangchao1230 — DISMISSED** [review]

File: scripts/runtime_mgmt/clean-disk-flow/flow.dag.yaml (hunk 3 above)
Comment (zhengfeiwang) [reply scripts/runtime_mgmt/clean-disk-flow/flow.dag.yaml]: removed, and added `schema`

**Pull request merged**